README.md

```diff
@@ -156,6 +156,8 @@ It supports these props, while passing any others through to the `children`:
 |scope|PropTypes.object|Accepts custom globals that the `code` can use
 |noInline|PropTypes.bool|Doesn’t evaluate and mount the inline code (Default: `false`). Note: when using `noInline` whatever code you write must be a single expression (function, class component or some `jsx`) that can be returned immediately. If you'd like to render multiple components, use `noInline={true}`
 |transformCode|PropTypes.func|Accepts and returns the code to be transpiled, affording an opportunity to first transform it
+|onError|PropTypes.func|Called with `Error` object when render fails. Allows you to listen for errors and implement custom logic.|
+|onRender|PropTypes.func|Called when component is rendered successfully. Can be used alongside `onError` to clear own error state.|
```

Comment:
I wonder if we could use `onError` for this same purpose? If `onRender` is called really only when render is called successfully (presumably after error) could we instead just call `onError` again with the default state such as `{ error: undefined }`. If I understand the point of this prop correctly I could imagine just resetting the error state would be sufficient for clearing any external error states?

Reply:
Sure, that's definitely doable!

I'm thinking it might be a _tad_ confusing to call it `onError` when it doesn't explicitly emit errors though. Maybe having the function signature be `onRender(error?: Error) => void` would make more sense? From first glance you'd be able to see what the event is about, and that it can emit both error and non-error state.

Let me know what you'd prefer and I'll make the updates. 🙂

Reply:
How about `onChange(code: string, error?: Error)`